doc/api/esm.md

```diff
@@ -1272,11 +1291,18 @@ export async function getSource(url, context, defaultGetSource) {
 }
 ```
 
-#### <code>transformSource</code> hook
+#### `transformSource(source, context, defaultTransformSource)`
 
 > Note: The loaders API is being redesigned. This hook may disappear or its
 > signature may change. Do not rely on the API described below.
 
+* `source` {string|SharedArrayBuffer|Uint8Array}
+* `context` {Object}
+  * `format` {string}
+  * `url` {string}
+* Returns: {Object}
+  * `source` {string|SharedArrayBuffer|Uint8Array}
```

Comment:
Same question as above —  is this supposed to be documented as an object wrapped in a promise (compare the JSDoc below)?